drivers/storage/portworx/portworx.go

```diff
@@ -499,6 +499,10 @@ func (p *portworx) preflightShouldRun(toUpdate *corev1.StorageCluster) bool {
 			return true
 		}
 
+		if preflight.IsAzure() { // Preflight runs on Azure & PX 3.1.0 or above
```

Comment:
nit: `& PX 3.1.0 or above` can be removed from the comment. It's repetitive as line 497 already covers that.